lib/ansible/plugins/callback/unixy.py

```diff
@@ -236,6 +236,6 @@ def v2_playbook_on_start(self, playbook):
 
     def v2_runner_retry(self, result):
         msg = "  Retrying... (%d of %d)" % (result._result['attempts'], result._result['retries'])
-        if self._run_is_verbose(result, verbosity=2):
+        if self._display.verbosity > 1:
```

Comment:
```suggestion
        if self._run_is_verbose(result):
```

Reply:
Agree with this suggestion, thank you for submitting the patch @jpic

Reply:
Done !
Note that line 227 there is self._display.verbosity > 3
https://github.com/ansible/ansible/pull/58801/files#diff-bfc0fb65ff63d5ae485526877e282f97R227
Have a great day